app/uk/gov/hmrc/eeittadminfrontend/controllers/GformsController.scala

```diff
@@ -39,32 +40,39 @@ class GformsController(val authConnector: AuthConnector)(implicit appConfig: App
       formWithErrors => {
         Future.successful(BadRequest(uk.gov.hmrc.eeittadminfrontend.views.html.gform_page(gFormForm)))
       },
-      gformIdAndVersion =>
+      gformIdAndVersion => {
+        Logger.info(s" ${request.session.get("token").get} Queried for ${gformIdAndVersion.formTypeId} ${gformIdAndVersion.version}")
         GformConnector.getGformsTemplate(gformIdAndVersion.formTypeId, gformIdAndVersion.version).map {
           case Some(x) => Ok(Json.prettyPrint(x))
           case _ => Ok("Error or does not exist")
         }
+      }
     )
   }
 
   def saveGformSchema = Authentication.async(parse.urlFormEncoded) { implicit request =>
     val template = Json.parse(request.body.apply("template").mkString)
     GformConnector.saveTemplate(template).map {
       x =>
-        x.status match {
-          case 200 => Ok("Saved")
-          case _ => Ok("Something went wrong")
+        {
+          Logger.info(s" ${request.session.get("token").get} saved ID: ${template \ "formTypeId"} version: ${template \ "version"}")
+          x.status match {
+            case 200 => Ok("Saved")
+            case _ => Ok("Something went wrong")
```

Comment:
`GformConnector.saveTemplate(template).map (Ok("saved)))` should be enough and fine here.